specs/card.proto

```diff
@@ -30,11 +30,21 @@ message BavariaCardTypeExtension {
   optional BavariaCardType card_type = 1;
 }
 
+message StartDayExtension {
+  // Start Day (calculated from 1970-01-01) of the beginning of the validity period.
+  // Cards are always invalid on 1970-01-01 (because day = 0 is excluded).
+  // The calculated day is inclusive (e.g. if day == 1, then the card is valid on 1970-01-02, but not on 1970-01-03).
+  // Because the data type is unsigned, there are no dates before 1970-01-01.
+  // 2^32days = 11 759 230.8 years.
+  optional uint32 start_day = 1;
+}
+
 message CardExtensions {
   optional RegionExtension extension_region = 1;
   optional BirthdayExtension extension_birthday = 2;
   optional NuernbergPassNumberExtension extension_nuernberg_pass_number = 3;
   optional BavariaCardTypeExtension extension_bavaria_card_type = 4;
+  optional StartDayExtension extension_start_day = 5;
 }
```

Comment:
Please add a test that checks the generated hash of a cardinfo using this extension to stay stable.
Please add a test both to frontend and administration

Reply:
You mean because you could change the time/date on the local device to make the card valid?

Reply:
No I imagine a unit test that checks whether the hashed card info with the newly added extension stays stable over time (... for when we modify the code / protobuf in the future).

There are already some tests involving this hash stability, but another one with the added extension would be beneficial.

Reply:
Oh sry I wanted to reply to your first comment that it should be added to db. The rest was clear.